deploy.php

```diff
@@ -0,0 +1,300 @@
+ *************************/
+
+set('use_relative_symlink', false);
+
+// Shared files/dirs between deploys
+add('shared_files', ['.env.local', 'app/config/parameters.yml']);
+add('shared_dirs', ['src/Frontend/Files']);
+
+// Writable dirs by web server
+add('writable_dirs', []);
+
+// Disallow stats
+set('allow_anonymous_stats', false);
+
+// Composer
+set('composer_options', '{{composer_action}} --verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader --no-suggest');
+
+// Sentry
+set(
+    'sentry',
+    [
+        'organization' => get('sentry_organization'),
+        'projects' => [
+            get('sentry_project_slug'),
+        ],
+        'token' => get('sentry_token')
+    ]
+);
+
+/*****************
+ * Task sections *
+ *****************/
+// Upload tasks
+/**
+ * Install assets from public dir of bundles
+ * @Override from symfony.php - don't know if we need this for Fork
+ */
+task(
+    'deploy:assets:install',
+    function () {
+        $packageFile = file_get_contents('package.json');
+        $package = json_decode($packageFile, true);
+
+        if (!array_key_exists('theme', $package)) {
+            writeln(
+                [
+                    '<comment>No theme found in package.json</comment>',
+                ]
+            );
+
+            return;
+        }
+
+        runLocally('gulp build');
+
+        $theme = $package['theme'];
+        $remotePath = '{{release_path}}/src/Frontend/Themes/' . $theme . '/Core';
+
+        upload(__DIR__ . '/src/Frontend/Themes/' . $theme . '/Core/', $remotePath);
+    }
+)->desc('Generate and upload bundle assets');
+after('deploy:update_code', 'deploy:assets:install');
+
+/**
+ * Migrate database
+ * @Override from symfony.php which executes doctrine:migrations
+ */
+task('database:migrate', function () {
+    cd('{{deploy_path}}/shared/');
+
+    if (!test('[ -f executed_migrations ]')) {
+        run('touch executed_migrations');
+    }
+
+    $parameters = Yaml::parse(run('cat app/config/parameters.yml'))['parameters'];
+    $executedMigrations = explode("\n", run('cat executed_migrations'));
+
+    cd('{{release_path}}/migrations/');
+
+    $dirs = explode(',', run('find {{release_path}}/migrations/* -maxdepth 0 -type d | tr "\n" ","'));
+    foreach ($dirs as $dir) {
+        if (empty($dir)) {
+            continue;
+        }
+
+        $shortName = basename($dir);
+        if (in_array($shortName, $executedMigrations)) {
+            continue;
+        }
+
+        // TODO should probably back up database
+
+        if (test('[ -f ' . $shortName . '/update.sql ]')) {
+            echo 'Running update.sql for ' . $shortName . "\n";
+
+            run('mysql --default-character-set="utf8" --host=' . $parameters['database.host'] . ' --port=' . $parameters['database.port'] . ' --user=' . $parameters['database.user'] . ' --password=' . $parameters['database.password'] . ' ' . $parameters['database.name'] . ' < ' . $shortName . '/update.sql');
```

Comment:
Misschien interessant om zelfde manier te doen als in https://github.com/tijsverkoyen/deployer-sumo/blob/master/recipe/database.php#L99

Reply:
Eventueel wel een PR maken om die `--default=character-set` toe te voegen